bin/GameIndex.dbf

```diff
@@ -39981,6 +40126,16 @@ Serial = SLUS-21314
 Name   = Ruff Trigger - Vancore Conspiracy
 Region = NTSC-U
 Compat = 4
+EETimingHack = 1 // Needed to going ingame
+[patches = f73488d5]
+	comment=Patch by Prafull
+	//quick freeze fix
```

Comment:
You should fix these patch comments as well. Some are correct now, and others are not.